src/inc/stackstring.h

```diff
@@ -0,0 +1,196 @@
+// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+
+#ifndef __STACKSTRING_H_
```

Comment:
We already have SString in CoreCLR, cannot we use that instead? The StackString was originally created as a simplified version of the SString without all the dependencies that SString has.